CONTRIBUTING.md

```diff
@@ -301,3 +305,28 @@ Then in the `pybricks-micropython` directory:
     yarn build:debug
 
 [wasm]: https://webassembly.org/
+
+Build and deploy firmware
+-------------------------
+
+### Build the firmware
+
+Pick your Hub from the `bricks` sub-directory you want to compile.
+
+    poetry shell
+    make mpy-cross -j8
+    make -C bricks/primehub -j8
+       
+### Deploy the firmware to a hub
+
+First, follow the guide to turn your Hub into DFU mode:
+1. https://pybricks.com/install, or 
+2. https://dfu.pybricks.com for Spike Prime or Mindstorms Inventor
+                
+Execute the deployment:
+
+    make -C bricks/primehub -j8 deploy-dfu
```

Comment:
As Laurens pointed out, `deploy` should now work for all hubs. 
```suggestion
    make -C bricks/primehub -j8 deploy
```

Reply:
Will try it out, thanks.

Reply:
Tested and works. Great! Shall we remove the `deploy-dfu` target?

Reply:
Yes, it would probably be better to only have one way to do a thing.

Reply:
Done.

Reply:
Just a minuscule side-note: it could maybe confusing to write "turn into..." 
May I suggest to write "switch to.. "
It is clear from the instructions but if something fails for whatever reason, maybe people think that they have to do something else to "turn" the device into something else...

But thank you for that great addition. This whole project is making more and more fun by the day.

-Dietrich-

Reply:
Yep. Since the DFU mode does not apply to all hubs, I've re-worded the whole sentence to
> 1. Follow the guide to prepare your Hub for Pybricks firmware installation:

😆